### ISSUE TITLE
Issue importing/exporting sql dump file with tags table

### ISSUE THREAD

**Issue by seanhudson** [spatie/laravel-tags#35]
Perhaps this isn't a bug, but I'm having problems with the format of the tags table. I exported the entire site database in a sql dump. Then I tried to import that file into a staging database. The import goes well until it hits the tags table. Then this error is thrown.

`Cannot create a JSON value from a string with CHARACTER SET 'binary'.`

I haven't worked with JSON in MySQL before this package. So, I'm sure I'm missing something. Does anyone have experience with this?

**Comment by marcusmoore** [comment]
What database and versions are you running in development and staging?

It's also possible the character set might be different between the two.

**Comment by seanhudson** [comment]
@marcusmoore thanks for your reply. I _just_ found a solution to my problem.

I was using Sequel Pro to create the SQL dump files. It's my main database management tool. Anyway, the latest stable release of Sequel Pro apparently doesn't have full JSON support. So, when I was exporting a SQL dump it was not exporting properly.

So, I found this thread (http://stackoverflow.com/questions/38078119/mysql-5-7-12-import-cannot-create-a-json-value-from-a-string-with-character-set) that suggested a method to convert the binary string in the SQL export to a usable format. That worked. However, that wasn't quite a sustainable solution.

Then I found out that the Sequel Pro nightly build now supports JSON. I downloaded it, and it exports and imports properly.

So, for me problem solved. Maybe this will help other folks.

BTW, I'm on MySQL v 5.7.17 on both development and staging.
